client/package.json

```diff
@@ -68,6 +68,7 @@
     "@babel/preset-react": "^7.7.4",
     "@testing-library/jest-dom": "^5.0.2",
     "@testing-library/react": "^9.3.2",
+    "@types/testing-library__jest-dom": "^5.14.1",
```

Comment:
I believe this is meant to solve a problem you had testing. My jest-foo is lacking.

1. What problem did it solve?

<details>

My concern is the addition of a new dependency. That is akin to an app-wide change.

</details>

Reply:
I did install this in an attempt to debug, but ended up not being used. How would I walk back this install?

Reply:
Okie.

1. `npm uninstall --save-dev @types/testing-library__jest-dom`, which automatically:
	- removes this line
	- updates `package-lock.json`
2. Commit the change.